atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CQLKeyValueServices.java

```diff
@@ -152,7 +153,7 @@ public Void call() throws Exception {
                         if (!success) {
                             sb.append(" (retrieval timed out)");
                         }
-                        log.info(sb.toString());
+                        log.info("Query trace: {}", sb.toString());
```

Comment:
Generally should prefer `log.info("Query trace: {}", sb);` as the `toString()` is superfluous and in would incur copy overhead when INFO logging is not enabled. In this case we've already checked that info is enabled so there's a high likelihood that INFO logging will still be enabled by the time we hit this.